### PR TITLE
fix rename of object in socialobject commondocgenerator.class.php

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

**Comment by frederic34** [comment]
@eldy

File: htdocs/core/class/commondocgenerator.class.php [hunk 1]
```diff
@@ -236,7 +236,7 @@ public function get_substitutionarray_mysoc($mysoc, $outputlangs)
 			'mycompany_idprof5'=>$mysoc->idprof5,
 			'mycompany_idprof6'=>$mysoc->idprof6,
 			'mycompany_vatnumber'=>$mysoc->tva_intra,
-			'mycompany_object'=>$mysoc->object,
+			'mycompany_object'=>$mysoc->socialobject,
```
Comment (eldy) [inline htdocs/core/class/commondocgenerator.class.php:239]: Rename is done on an assignement only. Property is set only, not used by a read anywhere ?

Comment (frederic34) [reply htdocs/core/class/commondocgenerator.class.php:239]: https://github.com/Dolibarr/dolibarr/pull/23638/files/9bd3cb927dd3b3984c44c124baff58d7fb90da80#r1083589059

**Comment by frederic34** [comment]
https://github.com/Dolibarr/dolibarr/blob/857a3090cafa1916d88438e4cb4f8c3ad5154e20/htdocs/core/modules/dons/html_cerfafr.modules.php#L165
https://github.com/Dolibarr/dolibarr/blob/857a3090cafa1916d88438e4cb4f8c3ad5154e20/htdocs/societe/class/societe.class.php#L4367

**Pull request closed without merging**